Spanner/src/Transaction.php

```diff
@@ -81,6 +81,11 @@ class Transaction implements TransactionalReadInterface
      */
     private $isRetry = false;
 
+    /**
+     * @var array
+     */
+    private $options = [];
+
```

Comment:
This is already part of `TransactionalReadTrait`, so adding it here will not do anything
```suggestion
```